tests/unit_tests/audit_log/test_audit_table.py

```diff
@@ -21,3 +44,110 @@ def test_init():
 def test_empty_prefix():
     with pytest.raises(ValueError):
         AuditTable("my_schema", "")
+
+
+def test_x1_audit_query_no_subquery(audit_table, other_table):
+    other = other_table.fully_qualified
+    audit_query = AuditQuery(
+        audit_fields={BaseAuditColumns.EVENT_NAME.name.name: "my event"},
+    )
+    statement = next(audit_table.augment([audit_query]))
+    LOG.debug(f"insert statement: \n{statement}")
+    assert statement == cleandoc(
+        f"""
+        INSERT INTO {audit_table.name.fully_qualified} (
+          "EVENT_NAME",
+          "LOG_TIMESTAMP",
+          "SESSION_ID"
+        ) SELECT
+          'my event',
+          SYSTIMESTAMP(),
+          CURRENT_SESSION
+        """
+    )
+
+
+def test_audit_query_with_subquery(audit_table, other_table):
+    other = other_table.fully_qualified
+    select = SelectQueryWithColumnDefinition(
+        query_string=f"SELECT ERROR AS ERROR_MESSAGE FROM {other}",
```

Comment:
I think, later we need to provide the StageQueryHandler with the list of audit columns they can influence, that would make writing these queries easier

Reply:
See 
* #271